ipi/inputs/simulation.py

```diff
@@ -57,7 +57,7 @@ class InputSimulation(Input):
             InputRandom,
             {
                 "help": InputRandom.default_help,
-                "default": input_default(factory=Random),
+                "default": input_default(factory=(lambda: Random())),
```

Comment:
I don't think this is necessary if you don't hardcode a default seed
```suggestion
"default": input_default(factory=Random),
```

Reply:
I forgot to erase the lambda: